music21/stream/makeNotation.py

```diff
@@ -792,7 +801,11 @@ def makeRests(
 
     Changed in v6 -- all but first attribute are keyword only
 
-    Changed in v7 -- `inPlace` defaults False.
+    Changed in v7:
+
+      - `inPlace` defaults False
+      - Recurses into parts, measures, voices
+      - Gave priority to `timeRangeFromBarDuration` over `refStreamOrTimeRange`
```

Comment:
btw -- I wonder if refStreamOrTimeRange can be removed -- is it ever used?

Reply:
If we remove it, we'll be putting a constraint that you can only make rests in measures and voices, not flat streams. I'm not keen on taking that away.

After some thought I think it's time to default timeRangeFromBarDuration True. Especially since now we've given it priority.

Reply:
Nope, on even more thought, it's too sensitive to turn this on, because now that we recurse, we will descend into every part and measure and voice and complete every voice, filling measures with all kinds of unwanted rests. Better to leave how currently configured -- in the musicxml parsers, measures should get timeRangeFromBarDuration=True in the musicxml parsers, voices should not, which is what we have without adding more changes here.